main/src/main/java/org/mobilitydata/gtfsvalidator/validator/TripAndShapeDistanceValidator.java

```diff
@@ -98,15 +121,59 @@ static class TripDistanceExceedsShapeDistanceNotice extends ValidationNotice {
     /** The faulty record's shape max distance traveled. */
     private final double maxShapeDistanceTraveled;
 
+    /** The distance in meters between the shape and the stop. */
+    private final double distance;
+
     TripDistanceExceedsShapeDistanceNotice(
         String tripId,
         String shapeId,
         double maxTripDistanceTraveled,
-        double maxShapeDistanceTraveled) {
+        double maxShapeDistanceTraveled,
+        double distance) {
```

Comment:
Thoughts on defining this as geoDistanceToShape to match column in https://gtfs-validator.mobilitydata.org/rules.html#stop_too_far_from_shape-rule?